tests/Label/LabelTest.php

```diff
@@ -254,4 +256,106 @@ public function it_does_not_make_a_label_private_when_already_private(): void
             })
             ->then([]);
     }
+
+    /**
+     * @test
+     */
+    public function it_can_exclude_an_included_label(): void
+    {
+        $this->scenario
+            ->withAggregateId($this->uuid->toString())
+            ->given([$this->created])
+            ->when(function ($label) {
```

Comment:
Maybe type-hint the parameter of the anonymous function? I also think that a fat arrow method is possible.